Pod/Classes/CDTQQuerySqlTranslator.m

```diff
@@ -394,6 +413,18 @@ + (CDTQSqlParts *)wherePartsForAndClause:(NSArray *)clause usingIndex:(NSString
                           parameters:sqlParameters];
 }
 
++ (NSString *)placeholdersForInList:(NSArray *)values withParameters:(NSMutableArray *)sqlParameters
+{
+    NSMutableArray *inOperands = [NSMutableArray array];
+    for (NSObject *value in values) {
+        [inOperands addObject:@"?"];
+        [sqlParameters addObject:value];
+    }
+    
+    NSString *joined = [inOperands componentsJoinedByString:@", "];
+    return [NSString stringWithFormat:@"( %@ )", joined];
+}
```

Comment:
It's confusing for a method to both return a value and modify one of its operands. I'm therefore not sure this method clarifies what's going on. You could perhaps change `withParameters` to be `updatingParameterValues` to make this clearer, or just not separate out the method if it's only called once.